django/test/utils.py

```diff
@@ -640,3 +640,14 @@ def setUp(self):
 
     def tearDown(self):
         self.logger.handlers[0].stream = self.old_stream
+
+
+def tag(*tags):
+    """
+    Decorator to add tags to test
+    """
+    def decorator(function):
```

Comment:
@Stranger6667 decorator is not meant to work with whole classes but can be used like this. IMO setting tags explicitly as class attribute is better than decorating it.